package.json

```
@@ -150,6 +150,10 @@
         "category": "ONE",
         "icon": "$(x)"
       },
+      {
+        "command": "one.toolchain.runCfg",
+        "title": "Run"
```

```suggestion
        "command": "one.toolchain.runCfg",
        "title": "Run",
        "category": "ONE"
```
I think category missed?

---

The `category` does not need in this case. This is because it is not be provided in the command palette. :)